packages/server/utils/publish.ts

```diff
@@ -1,7 +1,7 @@
 import getPubSub from './getPubSub'
 
 export interface SubOptions {
-  mutatorId?: string
+  mutatorId: string | null // if the viewer's client has received a mutation response from the server, but a subsequent update happens in the mutation, update the client by publishing a subscription. To do this, set mutatorId to null/undefined: https://github.com/ParabolInc/parabol/blob/88a801d80d0c51c38b6c9722dfa80fbca8f7bebd/packages/server/graphql/ResponseStream.ts#L26
```

Comment:
+1 Sorry, I missed that it was optional already 🤦
I suggest making it optional again, adding a simpler comment like: "Passing the user id of the mutator will omit sending a message to that user" or similar. Then we can keep this change smaller.

Reply:
And remove the `| null` again